setup-kicad-library-utils/action.yml

```diff
@@ -0,0 +1,53 @@
+name: Set up KiCad Library Utils
+description: |
+  Installs KiCad Library Utils to the runner environment.
+
+  For more information, see:
+  https://gitlab.com/kicad/libraries/kicad-library-utils
+
+inputs:
+  version:
+    description: |
+      Git ref (branch, tag, or commit) of KiCad Library Utils to be installed
+    default: master
+
+runs:
+  using: "composite"
+
+  steps:
+    - name: Install dependencies
+      shell: bash
+      run: |
+        sudo apt-get update
+        sudo apt-get install -y git python3 python3-pip
+
```

Comment:
We should remove this. Either the system already has Python, or the user explicitly installs and choose its desired version.